src/__tests__/fireEvent.test.tsx

```diff
@@ -274,6 +275,44 @@ test('should not fire on non-editable TextInput with nested Text', () => {
   expect(onChangeTextMock).not.toHaveBeenCalled();
 });
 
+test.only('should not fire onFocus on non-editable TextInput wrapped', () => {
```

Comment:
@TMaszko pls remove focussed test, as it disables all other tests in the file.

Reply:
Yeah that is a leftover !